front/src/router/index.ts

```diff
@@ -298,6 +299,10 @@ const routes: Array<RouteConfig> = [
     component: Admin,
     redirect: ROUTES_PATH.ADMIN + ROUTES_PATH.USER,
     props: true,
+    meta: {
+      needAuthentication: true,
+      requiredRoles: [USER_PROFILE.SUPERADMIN, USER_PROFILE.ADMIN, USER_PROFILE.CREATOR],
```

Comment:
The `CREATOR` is not allowed to access these routes. Only `ADMIN` / `SUPERADMIN`.